src/createStreamMiddleware.ts

```diff
@@ -102,7 +103,8 @@ export default function createStreamMiddleware(options: Options = {}) {
   function processResponse(res: PendingJsonRpcResponse<unknown>) {
     const context = idMap[res.id as unknown as string];
     if (!context) {
-      throw new Error(`StreamMiddleware - Unknown response id "${res.id}"`);
+      console.warn(`StreamMiddleware - Unknown response id "${res.id}"`);
+      return;
```

Comment:
(could) Document why this is just a warning, not an error with a long comment

Reply:
done